indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -38,13 +52,8 @@
         // Cache for data of already visited categories
         // Shared between instances
         _cache: {},
-
-        _typeaheadTemplate:
-        '<div class="typeahead__container">\
-            <div class="typeahead__field">\
-                <span class="typeahead__query"></span>\
-            </div>\
-        </div>',
+        _delaySearchResults: null,
+        _currentRequest: null,
```

Comment:
This is also shared between instances and can lead to unwanted behavior in case we have two widgets. It's unlikely but I think we should initialize it on `_create` instead.